### ISSUE TITLE
Wildfly 12 does not serve WebJAR resources from Single WAR

### ISSUE THREAD

**Issue by jreznot** [cuba-platform/cuba#952]
### Environment

- Platform version: 6.9.0
- Client type: Web
- App server: Wildfly 12
- OS: Windows 10

### Description of the bug or enhancement

1. Build Single WAR
2. Deploy
3. Open web app

Result:
```
Caused by: java.lang.IllegalArgumentException: jquery.min.js could not be found. Make sure you've added the corresponding WebJar and please check for typos.
	at com.haulmont.cuba.web.sys.WebJarResourceUtils.throwNotFoundException(WebJarResourceUtils.java:163)
	at com.haulmont.cuba.web.sys.WebJarResourceUtils.getFullPath(WebJarResourceUtils.java:123)
	at com.haulmont.cuba.web.sys.WebJarResourceUtils.getWebJarPath(WebJarResourceUtils.java:63)
	at com.haulmont.cuba.web.sys.CubaBootstrapListener.getWebJarResource(CubaBootstrapListener.java:69)
	at com.haulmont.cuba.web.sys.CubaBootstrapListener.modifyBootstrapPage(CubaBootstrapListener.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.vaadin.event.ListenerMethod.receiveEvent(ListenerMethod.java:510)
	... 66 more
```

See also: https://www.cuba-platform.ru/discuss/t/problema-s-razvorachivaniem-na-wildfly-12/1777/7

**Comment by jreznot** [comment]
Relates to #944

**Comment by jreznot** [comment]
Depends on https://github.com/cuba-platform/cuba-gradle-plugin/issues/44

**Comment by jreznot** [comment]
Fixed in #965
